packages/eas-json/src/utils.ts

```diff
@@ -73,4 +93,36 @@ export class EasJsonUtils {
     const easJson = await accessor.readAsync();
     return resolveSubmitProfile({ easJson, platform, profileName });
   }
+
+  public static getCacheSettings<T extends Platform>(
+    buildProfile: BuildProfile<T>,
+    ctx: any
```

Comment:
```suggestion
    ctx: BuildContext<T>
```
Please try not to type things as `any` in TypeScript.

Reply:
@szdziedzic The problem is `BuildContext` comes from `eas-cli`, so importing it causes a circular dependency. I would have to move it to `eas-json` as well

Reply:
If `ctx.clearCache` is the only property you need to access from `ctx`, you can change this line to:
```suggestion
    { clearCache }: { clearCache: TYPE }
```